### PR TITLE
Added opportunity to set fixed amount of promo codes

### PR DESCRIPTION
**Pull request by OleksiiBrylin**
Now you can do :
Promocodes::create($amount, $reward, $data, $expiresIn, false, $amount);
$amount is an integer. 

You need to make migration if you have already installed laravel-promocodes
`php artisan vendor:publish`
`php artisan migrate`

### CODE REVIEW

File: migrations/2018_10_22_221099_add_amount_codes_row.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class AddAmountCodesRow extends Migration
```
[the remark this answers is not in the document]
Comment (OleksiiBrylin) [reply migrations/2018_10_22_221099_add_amount_codes_row.php]: I did the same as 2016_05_17_221000_create_promocodes_table.php

**Comment by OleksiiBrylin** [comment]
Fix indents

**Comment by zgabievi** [comment]
Great feature. I will write tests for this and then I will definitely merge ;)

**Comment by vonec** [comment]
i think this commit is to have a functionality of having the coupon valid for the first N uses ... correct me if im wrong .. hope this makes it to the repo ... rather than having a separae field it would be good to use the additional data to have [ maximum_uses => N , maximum_uses_per_user => X ] and work round that, i think this is one of the common uses cases

**Comment by vonec** [comment]
in addition to the above post it would be super easy if we can add another attribute which would cap the max promo credit by default where in if the percentage bonus is greater than Y and caps it to Y that pretty much wraps all use cases 

 [ maximum_uses => N , maximum_uses_per_user => X, maximum_bonus => Y ]

**Comment by vonec** [comment]
also we can add eligibility check to for the coupon for cases where coupon is valid only for a purchase of Z

[ maximum_uses => N , maximum_uses_per_user => X, maximum_bonus => Y, eligibility => [conditions] ]

**Comment by zek** [comment]
Any news?

**Pull request merged**